news/2024/2024-05-29-osu-catch-world-cup-2024-round-of-32-recap.md

```diff
@@ -0,0 +1,94 @@
+
+The Netherlands took an early 2-1 lead and were on the brink of winning the next two maps as well. However, on both [MM2](https://osu.ppy.sh/beatmapsets/2184097#fruits/4615639) and [NM3](https://osu.ppy.sh/beatmapsets/2184033#fruits/4615512), China were able to snatch victory from the jaws of defeat as they overtook the Netherlands twice in the ending sections of both maps. Not too demoralized by their two losses, the Netherlands tied things up at 3 a piece going into the last pick of each team. China were able to win on their [NM4](https://osu.ppy.sh/beatmapsets/2184055#osu/4615558) pick even when the Dutch star player [b-a-d-s123](https://osu.ppy.sh/users/3656717) SS’ed the pick. Going into the final pick of the match, the Netherlands opted to go [DT1](https://osu.ppy.sh/beatmapsets/2184044#fruits/4615540) and drafted a roster of [Dutchman](https://osu.ppy.sh/users/10552580), [Sartan](https://osu.ppy.sh/users/4100941) and b-a-d-s123. They start off really well and build a substantial lead of 13.000 points going into the last 20 seconds of the map. They’ve lost MM2 in the ending sections of the map, they’ve lost NM3 when it was already in the bag, surely they won’t lose now, right?
+
+In a absolutely dramatic turn of events, a single drop from the Dutch team overturns their lead in the final seconds of the map, giving China a 5th point on the board and securing them the victory. Heartbreak for the Dutchies.
+
+### ::{ flag=AR }:: Argentina vs. ::{ flag=DK }:: Denmark
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2155904903&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+To close out the weekend on Sunday, seed 7 Argentina took on seed 26 Denmark. Argentina, fielding the highest ranked player in the entire tournament [-Story-](https://osu.ppy.sh/users/8172283), were caught by surprise by team Denmark as Denmark was able to convert the first two maps into two points. Denmark found a surprising weakness in the Mixed Mod bracket on the Argentinian team and utilized it to perfection. However, before the third map started, the veteran for the Argentinian team [VelperK](https://osu.ppy.sh/users/348467) joined the lobby. VelperK’s first appearance in the osu!catch world cup dates back to 2013 and he used this experience to absolutely obliterate the entire lobby for the next 30 minutes. He started off with a full combo on [NM1](https://osu.ppy.sh/beatmapsets/2184377#fruits/4617105), then proceeded to SS [HR1](https://osu.ppy.sh/beatmapsets/2183882#fruits/4615165), before continuing on and full combo-ing the remaining three maps to close out the match 5-2.
+
+## Round statistics
+
+We end this recap with a couple of interesting statistics to share:
```

Comment:
usually, unless necessary, "we" or "i" are not used as we are not trying to directly speak to the reader

```suggestion
To top this week's recap off, here are a couple of interesting statistics to share:
```